### PR TITLE
Custom DC: Wrap terraform commands in a script

### PR DESCRIPTION
**Pull request by Fructokinase**
Wrap setup and installation of DC terraform commands inside one script.

### CODE REVIEW

File: scripts/get-dc.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+echo "Installing Custom Datacommons web application in $PROJECT_ID."
+
+# Clone DC website repo and mixer submodule.
+git clone https://github.com/datacommonsorg/website
+git submodule foreach git pull origin master
+git submodule update --init --recursive
+
+cd website/deploy/terraform-datacommons-website/examples/setup
+
+terraform init && terraform apply \
```
Comment (shifucun) [inline scripts/get-dc.sh:40]: is terraform in available out of the box?

**Review by shifucun — APPROVED** [review]
very nice!!

File: scripts/get-dc.sh (hunk 1 above)
Comment (Fructokinase) [reply scripts/get-dc.sh:68]: It is on cloud shell (one of the benefits), but I should not assume that. Added a checker for gcloud and terraform commands.

Comment (shifucun) [reply scripts/get-dc.sh:68]: That's nice so this can run everywhere

**Pull request merged**